identity-account/src/types/identity_setup.rs

```diff
@@ -4,9 +4,11 @@
 use identity_core::crypto::PrivateKey;
 
 /// Configuration used to create a new Identity.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
 pub struct IdentitySetup {
   /// Use a pre-generated Ed25519 private key for the DID.
+  // TODO: Quick hack to get `IdentitySetup` to implement Serialize, by always using `None`
```

Comment:
Leftover TODO?

Reply:
This is needed for the `RemoteAccount` and I currently see that more as an example of a synchronous protocol that we or our users might want to build at some point. We don't really have to merge that in with this PR. I suppose a better approach would then be to duplicate the `IdentitySetup` so it doesn't affect the existing types in crates outside `identity-actor`.

Reply:
Removed the TODO and cloned the type for the `RemoteAccount` instead.